jsx/App/locale/LocaleConstants.jsx

```diff
@@ -0,0 +1,210 @@
+// Put language constants here to use in your display.
```

Comment:
I think a good way to do this would be just moving the text for the main pages back into the component files. So, for `AboutPage`, `GlossaryPage`, and `LandingPage` the text would be defined in the same file as the component (example below). Do you think this would be approachable enough? The component files are already clearly named and all exist at the first level directory, so they should be easy to find. & this way, the `constants` file only contains the shorter strings that are throughout the site. 


```
// GlossaryPage.jsx
import React from 'react';
import { TranslatableText } from './locale/TranslatableText.jsx'

// Glossary page content, can be formatted however one pleases using JSX.

const glossaryPageJSX = {
  [ENGLISH] :
    <div>
      <p>Input glossary here :) </p>
      <p>To customize this welcome text, edit the jsx/App/LocaleConstants.jsx file, then run webpack for your changes to take effect. </p>
    </div>,
  [ESPANOL] :
    <div>
      <p>Ingrese el glosario aquí :) </p>
      <p>Para personalizar este texto de bienvenida, edite el archivo 'jsx/App/LocaleConstants.jsx' y luego ejecute el paquete web para que los cambios surtan efecto. </p>
    </div>,
  [FRANCAIS] :
    <div>
      <p>Saisissez le glossaire ici :) </p>
      <p>Pour personnaliser ce texte de bienvenue, modifiez le fichier 'jsx/App/LocaleConstants.jsx', puis exécutez webpack pour que vos modifications prennent effet. </p>
    </div>,
};

export const GlossaryPage = () => <TranslatableText dictionary={glossaryPageJSX} />;
```

Reply:
Yes! Putting the text in the component (e.g. GlossaryPage.jsx) is how we should do it. (I know we discussed this option earlier and I said to put everything in a single file... sorry for waffling!)

Reply:
Done!